src/ol/style/Icon.js

```diff
@@ -43,6 +43,8 @@ import {getUid} from '../util.js';
  * @property {Array<number>} [displacement=[0, 0]] Displacement of the icon in pixels.
  * Positive values will shift the icon right and up.
  * @property {number} [opacity=1] Opacity of the icon.
+ * @property {number} [width] The width of the icon in pixels. This can't be used together with `scale` or `size`.
+ * @property {number} [height] The height of the icon in pixels. This can't be used together with `scale` or `size`.
```

Comment:
```suggestion
 * @property {number} [width] The width of the icon in pixels. This can't be used together with `scale`.
 * @property {number} [height] The height of the icon in pixels. This can't be used together with `scale`.
```

Reply:
Comment still applies.